mp/importer/batcher.py

```diff
@@ -26,9 +26,42 @@ def run_in_batches(iterable, end_batch_callback=None, batch_size=2, batch_start=
             if position_in_batch == 0: 							# ends completed batches
                 end_batch_callback()
                 ended = True
-        if max_batches is not None and current_batch == batch_start + max_batches:					# if gets maximum allowed batches, break 
+        if max_batches is not None and current_batch == batch_start + max_batches:	# if gets maximum allowed batches, break 
             break
     if ended is False:
         end_batch_callback()	
 
 
+def parse_arguments(args=None):		 						# receives a list of all the arguments and create a parser with only batcher args	
```

Comment:
I think that the API is wrong here. In general you will want to use the batcher as part of a larger script, not by itself. The way that this API is constructed, it is impossible to use it as part of a larger script as the script has no easy access to the parser to add other arguments.

I would prefer to see this API:
```
def add_arguments(parser):
```
so that the parser can be created/modified outside the function.